addons/mail/static/src/components/thread_view/thread_view.xml

```diff
@@ -10,6 +10,15 @@
                     </div>
                 </t>
                 <t t-elif="threadView.threadCache.isLoaded or threadView.thread.isTemporary">
+                    <t t-if="threadView.nonEmptyMessages.length === 0 and (threadView.thread.activities.length or threadView.threadViewer and threadView.threadViewer.chatter and threadView.threadViewer.chatter.isAttachmentBoxVisible)">
+                        <div class="o_MessageList_empty_separator">
+                            <hr class="o_MessageList_empty_separatorLine"/>
+                            <span class="o_MessageList_empty_separatorText">
+                                Messages
+                            </span>
+                            <hr class="o_MessageList_empty_separatorLine"/>
+                        </div>
+                    </t>
```

Comment:
Also don't forget to update structural classnames when you move the LOCs from one component to another. It was in `MessageList`, right now it's in `ThreadView`. When you apply changes from my previous comment, this will be in `Chatter` component.